Formula/ddcctl.rb

```diff
@@ -0,0 +1,16 @@
+class Ddcctl < Formula
+  desc "DDC monitor controls (brightness) for Mac OSX command-line"
+  homepage "https://github.com/kfix/ddcctl"
+  url "https://github.com/kfix/ddcctl/archive/refs/tags/v0.tar.gz"
+  sha256 "8440f494b3c354d356213698dd113003245acdf667ed3902b0d173070a1a9d1f"
+  license :public_domain
+
+  def install
+    bin.mkpath
+    system "make", "install", "INSTALL_DIR=#{bin}"
+  end
+
+  test do
+    assert_match "found 0 external display", shell_output("#{bin}/ddcctl -d 1 -b 100", 1)
```

Comment:
```suggestion
    output = shell_output("#{bin}/ddcctl -d 1 -b 100", Hardware::CPU.arm? ? 255 : 1)
    assert_match(/found \d external display/, output)
```
Try this.

Reply:
Oh, and please squash all your commits together after you make these changes (along with the `depends_on :macos`).